yb-voyager/test/utils/testutils.go

```diff
@@ -424,3 +424,10 @@ func CreateTempFile(dir string, fileContents string, fileFormat string) (string,
 
 	return file.Name(), nil
 }
+
+func CreateNameTuple(s string, defaultSchema string, dbType string) sqlname.NameTuple {
```

Comment:
nit: Since you're only populating the sourceName, indicate that somehow in the function name.